openmls/src/group/core_group/new_from_welcome.rs

```diff
@@ -30,9 +30,16 @@ impl CoreGroup {
             key_package_bundle.key_package.leaf_node().encryption_key(),
         )
         .ok_or(WelcomeError::NoMatchingEncryptionKey)?;
-        leaf_keypair
+
+        // Delete the [`KeyPackage`] and the corresponding private key from the
+        // key store, but only if it doesn't have a last resort extension.
```

Comment:
Could you change the text here (and in the debug log entry below) to indicate that this is about the leaf encryption key pair and not the KeyPackage?

Reply:
Thanks!